src/Cookie.php

```diff
@@ -37,91 +58,103 @@ final class Cookie
     /**
      * @var string value of the cookie
      */
-    private string $value;
+    private string $value = '';
```

Comment:
It shouldn't have default value

Reply:
Yep, default value here can be omitted as constuctor sets it to default empty string anyway. Empty string is used instead of previous required value, as cookie can have empty value according to RFC.